docs/dev/table/connectors/filesystem.md

```diff
@@ -65,6 +65,9 @@ CREATE TABLE MyUserTable (
 
 <span class="label label-danger">Attention</span> File system sources for streaming is still under development. In the future, the community will add support for common streaming use cases, i.e., partition and directory monitoring.
 
+<span class="label label-danger">Attention</span> The behaviour of file system connector is much different from previous one: 
```

Comment:
previous one -> `previous legacy filesystem connector`

Reply:
Thanks. I will fix it.